openpype/hosts/blender/plugins/publish/integrate_blend.py

```diff
@@ -0,0 +1,129 @@
+from concurrent.futures import Future, ThreadPoolExecutor
+from functools import partial
+import subprocess
+
+import bpy
+from bson.objectid import ObjectId
+import pyblish
+
+from openpype.hosts.blender.api.utils import BL_TYPE_DATAPATH
+from openpype.hosts.blender.utility_scripts import (
+    make_paths_relative,
+    update_representations,
+)
+from openpype.modules.base import ModulesManager
+from openpype.plugins.publish.integrate_hero_version import (
+    IntegrateHeroVersion,
+)
+from openpype.settings.lib import get_project_settings
+
+
+class IntegrateBlenderAsset(pyblish.api.InstancePlugin):
+    label = "Integrate Blender Asset"
+    hosts = ["blender"]
+    order = IntegrateHeroVersion.order + 0.01
+
+    def process(self, instance):
+        # Check enabled in settings
+        project_entity = instance.data["projectEntity"]
+        project_name = project_entity["name"]
+        project_settings = get_project_settings(project_name)
+        host_name = instance.context.data["hostName"]
+        host_settings = project_settings.get(host_name)
+        if not host_settings:
+            self.log.info('Host "{}" doesn\'t have settings'.format(host_name))
+            return None
+
+        use_path_management = host_settings.get("general", {}).get(
+            "use_paths_management"
+        )
+
+        # Prepare pool for commands
+        pool = ThreadPoolExecutor()
+
+        # Get sites to sync with
+        manager = ModulesManager()
+        sync_server_module = manager.modules_by_name["sync_server"]
+        site = sync_server_module.get_remote_site(project_name)
+
+        # Get published and hero representations
+        representations = instance.data.get("published_representations")
+        representations.update(instance.data.get("hero_representations", {}))
+
+        # Run commands for all published representations
+        instance.data.setdefault("representations_futures", [])
+        for representation in representations.values():
+            representation = representation["representation"]
+            published_path = representation.get("data", {}).get("path")
+
+            # Set main command
+            main_command = [bpy.app.binary_path, published_path, "-b"]
+
+            # If not workfile, it is a blend and there is a published file
+            if representation.get("name") == "blend" and published_path:
```

Comment:
If condition has nothing in else nor underneath, so I would recommend to swap the condition to
```suggestion
            if representation.get("name") != "blend" or notpublished_path:
                continue
```